form.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title> VIN Formation</title>
+    <!-- <link rel="stylesheet" type="text/css" href="./assets/css/style.css" /> -->
+    <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400&family=Merriweather:wght@700&display=swap"
+    rel="stylesheet" />
+    <script
+    src="https://code.jquery.com/jquery-3.5.1.min.js"
+      integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+      
+    <div class="content">
+        <form id="search-form" class="form">
+            <h1>VIN Formation</h1>
+            <input type="text" id="input" placeholder="please enter VIN number" name="VIN" maxlength="100"/>
+            <button id="submit" type="submit">Submit</button>
+        </form>
+    </div>
+    <div id="VIN">
+    <h1>Showing results for <span id="type">vehicle type</span></h1>
+        <img>  </img>
+         <h2>vVehicle Model   </h2>
```

Comment:
it might be easier to group the info section in its own <div> that way we can just set that wrapper div to be hidden or shown depending on what is going on.

Reply:
again not sure why but the word div was left out of my above comment so it should say "it might be easier to group the info section in its own div"